### ISSUE TITLE
Color not automatically selected when using RGBA

### ISSUE THREAD

**Issue by CrashyBang** [xiaokaike/vue-color#177]
Colors like so:

```js
data() {
    return {
        colors: {
            rgba: {
                r: 255,
                g: 0,
                b: 0,
                a: 1,
            }
        }
    };
},
```

Input like so:
```
<chrome-picker :value="colors" @input="updateColor" :disableFields="true"/>
```

Everything renders fine and the input event works etc... the only issue is the picker circle/slider have not adjusted to represent the initial `colors` value, however when I use a Hex string (i.e `#F00`) this does happen.

**UPDATE**

It actually  only seems to not be working when I try to update `this.colors` like so:

```js
const colors = node.fill.match(/\d+/g); // Standard rgba string

this.colors = {
    rgba: {
        r: +colors[0],
        g: +colors[1],
        b: +colors[2],
        a: +colors[3]
    }
};
```

I take it this is not reactive intentionally...?

Cheers.

**Comment by egeersoz** [comment]
I am also wondering about this.

**Comment by terrymun** [comment]
I think that is due to VueJS limitations in detecting mutations in nested object properties.

**Comment by Vercoutere** [comment]
> I think that is due to VueJS limitations in detecting mutations in nested object properties.

Works perfectly fine with hsl values though...

**Comment by linx4200** [comment]
Hi guys. After I checked, confirm that this is a long-standing bug. Will fix it in these two days.

**Comment by linx4200** [comment]
fixed version v2.7.1
